packages/beacon-node/src/network/processor/gossipHandlers.ts

```diff
@@ -177,28 +186,36 @@ function getDefaultHandlers(modules: ValidatorFnsModules, options: GossipHandler
     const blockHex = prettyBytes(blockRoot);
 
     const delaySec = chain.clock.secFromSlot(slot, seenTimestampSec);
-    const recvToVal = Date.now() / 1000 - seenTimestampSec;
+    const recvToValLatency = Date.now() / 1000 - seenTimestampSec;
 
     const {blockInput, blockInputMeta} = chain.seenGossipBlockInput.getGossipBlockInput(config, {
       type: GossipedInputType.blob,
       blobSidecar,
       blobBytes,
     });
 
-    metrics?.gossipBlob.receivedToGossipValidate.observe(recvToVal);
```

Comment:
this previously was incorrect because it was observing the latency not the actual rec to validation time, fixed below